project-description.md

```diff
@@ -0,0 +1,21 @@
+## Basic features
```

Comment:
move to `projects/python/<your project name>.md` or whatever language you're using